aptos-move/framework/aptos-framework/sources/account.move

```diff
@@ -428,6 +429,35 @@ module aptos_framework::account {
         (signer, signer_cap)
     }
 
+    /// Create a resource account using the vectorized Unix timestamp
```

Comment:
nit: Too many line breaks here. Can we make the lines as long as possible without running over the char limit?

Reply:
@movekevin line breaking increased to 100 characters per [e613916f62c55ad7d04de9e7198852ecad51ccaf](https://github.com/aptos-labs/aptos-core/pull/4173/commits/e613916f62c55ad7d04de9e7198852ecad51ccaf)